rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -218,45 +221,58 @@ public void didOpen(DidOpenTextDocumentParams params) {
 
     @Override
     public void didChange(DidChangeTextDocumentParams params) {
-        logger.trace("Change contents: {}", params.getTextDocument());
+        logger.debug("Did Change file: {}", params.getTextDocument());
```

Comment:
NP: we should make them trace again, to avoid spamming the log. as changes happen a lot.

Reply:
true, that's maybe better.